core/src/banking_stage.rs

```diff
@@ -1070,6 +1070,9 @@ impl BankingStage {
                 let p = &msgs.packets[*tx_index];
                 let tx: Transaction = limited_deserialize(&p.data[0..p.meta.size]).ok()?;
                 tx.verify_precompiles().ok()?;
+                let message_bytes = Self::packet_message(p)?;
+                let message_hash = Message::hash_raw_message(message_bytes);
+                let tx = SanitizedTransaction::try_create(Cow::Owned(tx), message_hash).ok()?;
```

Comment:
nice simplification! Do you think if we need need this block of code then? https://github.com/solana-labs/solana/blob/master/core/src/cost_model.rs#L201-L204

Reply:
No longer needed, I'll remove

Reply:
Removed